api/utils.py

```diff
@@ -218,23 +226,72 @@ def addWorkMeta(cls, work, **kwargs):
 
     @classmethod
     def formatEditionOutput(
-        cls, edition, records=None, showAll=False, formats=None, reader=None
+        cls, edition, records=None, dbClient=None, showAll=False, formats=None, reader=None
     ):
         editionWorkTitle = edition.work.title
         editionWorkAuthors = edition.work.authors
+        editionInCollection = cls.checkEditionInCollection(None, edition, dbClient)
         
         return cls.formatEdition(
-            edition, editionWorkTitle, editionWorkAuthors, records, formats, showAll=showAll, reader=reader
+            edition, editionWorkTitle, editionWorkAuthors, editionInCollection, records, formats, showAll=showAll, reader=reader
         )
-    
+
+    @classmethod
+    def checkEditionInCollection(cls, work, edition, dbClient):
+
+        collectionIDs = []
+
+        if work != None:
+            for edit in dbClient.session.query(Edition) \
+                .filter(Edition.work_id == work.id):
+                    for collectEdit in dbClient.session.query(COLLECTION_EDITIONS) \
+                        .filter(COLLECTION_EDITIONS.c.edition_id == edit.id):
+                            #Avoiding duplicate collection ids
+                            if collectEdit.collection_id not in collectionIDs:
+                                for collection in dbClient.session.query(Collection) \
+                                .filter(Collection.id == collectEdit.collection_id):
+                                    numOfItems = 0
+                                    for edit in collection.editions:
+                                        numOfItems += 1
```

Comment:
Could just be `len(collection.editions)` I think!

Reply:
That one line is equivalent to the for loop conditional I wrote so I'll replace it with your suggestion.